Dockerfile

```diff
@@ -1,18 +1,22 @@
-FROM rust as build
```

Comment:
Please don't overwrite existing Dockerfiles - intsead, create something like `Dockerfile.mayhem` to build the image for your fuzzing target, and restore the original Dockerfile.
You can reference the custom Dockerfile like so:
```
      - name: Build and push Docker image
        uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
        with:
          context: .
          push: true
          tags: ${{ steps.meta.outputs.tags }}
          labels: ${{ steps.meta.outputs.labels }}
          file: my/custom/Dockerfile.mayhem
```